editor.planx.uk/src/@planx/components/MapAndLabel/test/mocks/Trees.ts

```diff
@@ -1,12 +1,19 @@
+import { Schema } from "@planx/components/shared/Schema/model";
+
 import { PresentationalProps } from "../../Public";
 import { Trees } from "../../schemas/Trees";
 
+const mockTreeSchema: Schema = {
+  ...Trees,
+  fields: Trees.fields.filter((field) => field.type !== "date"),
+};
```

Comment:
Per above comment, this would be better place to set `description` for testing-only please !